network/api/v1/api.go

```diff
@@ -53,6 +55,25 @@ func (a *Wrapper) Routes(router core.EchoRouter) {
 			return audit.StrictMiddleware(f, network.ModuleName, operationID)
 		},
 	}))
+
+	const diagnosticsBasePath = "/status/diagnostics/network"
```

Comment:
should be added to `docs/_static/monitoring/v1.yaml` so it can be found in the API documentation

Reply:
Not sure, that means we're going to spec it, thus having to maintain backwards compatibility?

Reply:
`/status/diagnostics` is on there too and we do not maintain backwards compatibility for that.

I added the monitoring API for discoverability of endpoints without the intention of it being fully specced. `/status`, `/health` and probably `/metrics` should probably not change, so maybe we need make a distinction between machine readable monitoring and programmer debugging convenience where the latter can be changed without notice?

Reply:
Will add it